test/morpho-blue/TestBlueSnippets.sol

```diff
@@ -112,72 +107,82 @@ contract TestIntegrationSnippets is BaseTest {
         assertEq(expectedTotalBorrow, morpho.totalBorrowAssets(id));
     }
 
-    function testBorrowAPY(Market memory market) public {
-        vm.assume(market.totalBorrowAssets > 0);
-        vm.assume(market.totalSupplyAssets >= market.totalBorrowAssets);
+    function testBorrowAPY(uint256 amountSupplied, uint256 amountBorrowed, uint256 timeElapsed, uint256 fee) public {
+        vm.assume(fee < 1 ether);
+
+        _generatePendingInterest(amountSupplied, amountBorrowed, timeElapsed, fee);
+        morpho.accrueInterest(marketParams);
+
+        Market memory market = morpho.market(id);
+        uint256 borrowApyTrue = irm.borrowRate(marketParams, market).wTaylorCompounded(365 days);
+        uint256 borrowApyTest = snippets.borrowAPY(marketParams, market);
 
-        uint256 borrowTrue = irm.borrowRate(marketParams, market).wTaylorCompounded(1);
-        uint256 borrowToTest = snippets.borrowAPY(marketParams, market);
+        assertEq(borrowApyTrue, borrowApyTest, "Diff in snippets vs integration borrowAPY test");
 
-        assertEq(borrowTrue, borrowToTest, "Diff in snippets vs integration borrowAPY test");
+        if (borrowApyTrue > 0) {
```

Comment:
in which case `borrowApyTrue == 0`?

Reply:
in case amountBorrowed=0. 

note that it is killing a mutant in here!